frontend/src/metabase/actions/containers/ActionCreator/InlineDataReference.styled.tsx

```diff
@@ -4,8 +4,7 @@ import { color } from "metabase/lib/colors";
 import Button from "metabase/core/components/Button";
 import SidebarContent from "metabase/query_builder/components/SidebarContent";
 
-export const DataReferenceContainer = styled.div<{ isOpen: boolean }>`
-  display: ${props => (props.isOpen ? "block" : "none")};
```

Comment:
Now this component is conditionally render in JS, so no need to use `display: none` to hide it anymore.